dashboard/app/models/pd/application/application_base.rb

```diff
@@ -96,15 +96,15 @@ def set_type_and_year
       self.application_type = nil
     end
 
-    # Creates the following methods: accepted? incomplete? pending? unreviewed? waitlisted? pending_space_availability?
-    %w(accepted incomplete pending unreviewed waitlisted pending_space_availability).each do |attribute|
+    # Creates the following methods: accepted? incomplete? pending? unreviewed? waitlisted? pending_space_availability? awaiting_admin_approval?
+    %w(accepted incomplete pending unreviewed waitlisted pending_space_availability awaiting_admin_approval).each do |attribute|
       define_method(:"#{attribute}?") do
         status == attribute
       end
     end
 
     def set_applied_date
-      self.applied_at = Time.now if applied_at.nil? && unreviewed?
+      self.applied_at = Time.now if applied_at.nil? && (unreviewed? || awaiting_admin_approval?)
```

Comment:
You check or either `unreviewed?` or `awaiting_admin_approval?` a couple times. Would it be worth adding a helper method called something like `awaiting_action?` (my knowledge in this space is limited so feel free to wordsmith). It's also only a couple of places so it might not be worth it. Thoughts?

Reply:
Updated to `status_on_submit?` –– I'm sure we'll be using this more in the future. Good call!